config/psp/psp_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: controller-manager-psp 
```

Comment:
This sounds pretty generic to me. Should it be something more specific to not potentially conflict with something else?

Reply:
Check the diff in the description. The final rendered PSP has `capi-migration-controller-manager-psp` name.

With kustomize you specify global prefix added to all the names of the bases so you can keep all names and namespaces generic. This is really cool feature allowing to reuse bases between projects without any modificiations.